R/colourInput.R

```diff
@@ -194,59 +205,19 @@ colourInput <- function(inputId, label, value = "white",
 #' @export
 updateColourInput <- function(session, inputId, label = NULL, value = NULL,
                               showColour = NULL, palette = NULL, allowedCols = NULL,
-                              allowTransparent = NULL, transparentText = NULL,
+                              allowAlpha = NULL, allowTransparent = NULL, transparentText = NULL,
                               returnName = NULL) {
   message <- dropNulls(list(
-    label = label, value = formatHEX(value),
```

Comment:
I removed all of the formatHex stuff, since the javascript library now handles all of that natively